Tests/PHPCI/Build/BuilderTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace PHPCI\Build\Tests;
+
+use Symfony\Component\Yaml\Parser as YamlParser;
+
+class BuilderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testExecute_EmptyStageConvertNullToArray()
+    {
+        $builder = $this
+            ->getMockBuilder('PHPCI\Builder')
+            ->setMethods(null)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $config = (new YamlParser())->parse("stage:\n");
```

Comment:
Verifying the current source code, the `PHPCI\Plugin\Util\Executor::buildPlugin` is not expecting the second parameter as array, because of dd58dd68. I don't know why a plugin configuration must not always be an array, but, I'll remove my changes about it.